ni/measurements/metadata/v1/metadata_store_service.proto

```diff
@@ -94,6 +94,13 @@ service MetadataStoreService {
   rpc CreateAlias(CreateAliasRequest) returns (CreateAliasResponse);
   // Removes a registered alias
   rpc DeleteAlias(DeleteAliasRequest) returns (DeleteAliasResponse);
+
+  // Creates all of the metadata entries contained in the provided JSON document.
+  // The JSON document should conform to the schema defined by
+  // https://raw.githubusercontent.com/ni/ni-apis/main/ni/measurements/metadata/v1/registration.schema.json
+  // The response will contain all of associated the objects from the metadata store.  If the object
```

Comment:
nit: "all of associated the objects" => "all of the associated objects"

maybe also say "If the object already exists" rather than "If the object already existed"